test/bin/setup_volume_permissions.sh

```diff
@@ -1,10 +1,21 @@
 #!/usr/bin/env bash
+# grants circlci docker uid and gid required write permissions to project file system
 
 set -x
 
-sudo groupadd -g 3434 circleci_tinytds
-sudo usermod -a -G circleci_tinytds $USER
-sudo useradd circleci_tinytds -u 3434 -g 3434
-sudo usermod -a -G circleci_tinytds circleci_tinytds
-sudo chgrp -R circleci_tinytds .
+if $(dpkg --compare-versions $ruby_version lt 3.1)
```

Comment:
Hm, all of this is a bit confusing. Is it to prepare for Ruby 3.1?

Reply:
it was from the 3.1 work, yes. circleci changed the permissions in their container. i will stash this change for a future commit